src/components/TextArea.js

```diff
@@ -61,9 +61,7 @@ export const TextArea = ({
     <TextAreaContainer>
       <TextAreaBox
         value={value}
-        onChange={val => {
-          onChangeValue(val.target.value)
-        }}
+        onChange={val => onChange(val.target.value)}
         invalid={invalid || isLengthExceeded}
```

Comment:
@panjenny0 soo this isn't _exactly_ what you suggested, but I think I captured it in essence (ie. the element now uses `onChange`) 

The reason for `val.target.value` in the arrow function here is because <textarea>'s native onChange callback passes in an object (not a text value), so we have to use `val.target.value` in order to access the value string inside the <textarea> element

otherwise we get this haha
![ezgif com-gif-maker (2)](https://user-images.githubusercontent.com/47487758/99891122-56630080-2c1b-11eb-85c1-d1d862fb3065.gif)

anyway, minor difference imo, hope this is ok to pass review! :D